src/vma/dev/rfs.cpp

```diff
@@ -128,11 +128,15 @@ rfs::~rfs()
 		int counter = 0;
 		prepare_filter_detach(counter, true);
 		if (counter == 0) {
-			destroy_ibv_flow();
+			if (m_p_ring->get_type() == RING_SIMPLE) {
```

Comment:
m_p_ring->get_type() == RING_SIMPLE should move to destroy_ibv_flow( & create_ibv_flow